src/finitediff/timedomain/recordings.jl

```diff
@@ -22,7 +22,29 @@ struct Recordings{Ti<:Int64, Tv<:AbstractFloat}
 end
 
 """
-   constructor for recordings
+
+R = Recordings(rz, rx, params; <keyword arguments>)
+
+Constructs a Recordings structure.
+
+Please see the extended help with `?? Source`
+
+# Arguments
+
+-`rz::`: Receiver coordinate.
+-`rx::`: Receiver coordinate.
```

Comment:
Yes, do add that these are two vectors
```julia
-`rz::Vector`: ...
-`rx::Vector`: ...
```
I don't think all of them need to have that if they are obvious. We can discuss that later too.

Reply:
Ok! Yeah, for sure. I am asking because we need to make the package easy for users (and for us). In my experience, I have suffered from lack of understanding of types :(

Reply:
We can leave it without for now if it is confusing, but then notice that it would be like
```julia
-`rx`: Vector of ...
-`rz`: Vector of...
```
instead of 
```julia
-`rx::`: Vector of ...
-`rz::`: Vector of...
```
because the `::` constructor denotes type annotation (i.e., `a::Int` defines a as an integer).